utils/getInvoices/tests/getInvoices.maintenance.test.ts

```diff
@@ -6,108 +6,834 @@ import { ServiceType } from '@utils/constants'
-      expect.objectContaining({ type: ServiceType.Maintenance })
-    )
+      expect(invoices).toContainEqual(
+        expect.objectContaining({ type: ServiceType.Maintenance })
+      )
+    })
+    it('should NOT load when service = { rentPrice: 10 }, company = null', () => {
+      const service: Partial<IService> = {
+        rentPrice: 10,
+      }
+      const company: Partial<IRealestate> = null
+
+      const invoices = getInvoices({
+        service,
+        company,
+      })
+
+      expect(invoices).not.toContainEqual(
+        expect.objectContaining({ type: ServiceType.Maintenance })
+      )
+    })
+    it('should NOT load when service = { rentPrice: 10 }, company = { discount: 0 }', () => {
+      const service: Partial<IService> = {
+        rentPrice: 10,
+      }
+      const company: Partial<IRealestate> = {
+        discount: 0,
+      }
+
+      const invoices = getInvoices({
+        service,
+        company,
+      })
+
+      expect(invoices).not.toContainEqual(
+        expect.objectContaining({ type: ServiceType.Maintenance })
+      )
+    })
+    it('should load when service = { rentPrice: 10 }, company = { totalArea: 10 }', () => {
+      const service: Partial<IService> = {
+        rentPrice: 10,
+      }
+      const company: Partial<IRealestate> = {
+        totalArea: 10,
+      }
+
+      const invoices = getInvoices({
+        service,
+        company,
+      })
+
+      expect(invoices).toContainEqual(
+        expect.objectContaining({ type: ServiceType.Maintenance })
+      )
+    })
+    it('should load when service = { rentPrice: 0 }, company = { totalArea: 0 }', () => {
+      const service: Partial<IService> = {
+        rentPrice: 0,
+      }
+      const company: Partial<IRealestate> = {
+        totalArea: 0,
+      }
+
+      const invoices = getInvoices({
+        service,
+        company,
+      })
+
+      expect(invoices).toContainEqual(
+        expect.objectContaining({ type: ServiceType.Maintenance })
+      )
+    })
+    it('should load when service = { rentPrice: 10 }, company = { totalArea: 0 }', () => {
+      const service: Partial<IService> = {
+        rentPrice: 10,
+      }
+      const company: Partial<IRealestate> = {
+        totalArea: 0,
+      }
+
+      const invoices = getInvoices({
+        service,
+        company,
+      })
+
+      expect(invoices).toContainEqual({ 
+        type: ServiceType.Maintenance,
+        amount: 0,
+        price: 10,
+        sum: 0,
+      })
+    })
+    it('should NOT load when service = { rentPrice: 10 }, company = { totalArea: undefined }', () => {
+      const service: Partial<IService> = {
+        rentPrice: 10,
+      }
+      const company: Partial<IRealestate> = {
+        totalArea: undefined,
+      }
+
+      const invoices = getInvoices({
+        service,
+        company,
+      })
+
+      expect(invoices).not.toContainEqual(
+        expect.objectContaining({ type: ServiceType.Maintenance })
+      )
+    })
```

Comment:
change order to just like binary table for easier reading
- `should load when service = { rentPrice: 0 }, company = { totalArea: 0 }`
- `should load when service = { rentPrice: 0 }, company = { totalArea: 10 }`
- `should load when service = { rentPrice: 10 }, company = { totalArea: 0 }`
- `should load when service = { rentPrice: 10 }, company = { totalArea: 10 }`